File: 2292-counting-words-with-a-given-prefix/2292-counting-words-with-a-given-prefix.js
/**
 * @param {string[]} words
 * @param {string} pref
 * @return {number}
 */
var prefixCount = function(words, pref) {
    let cnt = 0;
    for (let st of words)
        if (st.startsWith(pref))
            cnt++;
    return cnt
};

File: 2292-counting-words-with-a-given-prefix/2292-counting-words-with-a-given-prefix.c
int prefixCount(char** words, int wordsSize, char* pref) {
    int i, cnt = 0;
    int len = strlen(pref);
    for (i = 0; i < wordsSize; i++) {
        if (strncmp(words[i], pref, len) == 0)
            cnt++;
    }
    return cnt;
}